blog.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>Est Laborum - Blog</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+
+ <!--        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/main.css"> -->
+    </head>
+    <body>
+      <header role="banner">
+        <h1>Est Laborum</h1>
+        <nav role="navigation">
+          <a href="index.html">Home</a>
+          <a href="contact.html">Contacts</a>
+        </nav>
+        <nav id="blognav" role="navigation">
+          <ul>
+            <li>
+              <a href="#Dolor">Dolor Sit Amet</a>
+            </li>
+            <li>
+              <a href="#Consectetur">Consectetur Adipiscing</a>
+            </li>
+            <li>
+              <a href="#Tempor">Tempor Incididunt</a>
+            </li>
+          </ul>
+        </nav
+      </header>
+      <main role="main">
+        <section>
+          <h1>Lorem Ipsum</h1>
+
+          <article id="Dolor" role="article">
+            <h2>Dolor Sit Amet</h2>
+            <h3>
```

Comment:
You've formatted your h2's and h3's different. Either way is totally fine, but consistency is key!